crates/bevy_render/src/render_resource/storage_buffer.rs

```diff
@@ -214,14 +223,18 @@ impl<T: ShaderType + WriteInto> DynamicStorageBuffer<T> {
         self.label.as_deref()
     }
 
+    pub fn set_usage(&mut self, usage: BufferUsages) {
+        self.buffer_usage.set(usage, true);
```

Comment:
I had to look up what the `.set` method does. It might be more intuitive to just use the `|` operator. But either one is fine.